framework/src/org/checkerframework/framework/util/FlowExpressionParseUtil.java

```diff
@@ -101,6 +101,26 @@
             .compile("^\"([^\"\\\\]|\\\\.)*\"$");
     /** Matches the null literal */
     protected static final Pattern nullPattern = Pattern.compile("^null$");
+    /** Matches an expression contained in matching start and end parentheses */
+    protected static final Pattern parenthesesPattern = Pattern.compile("^\\((.*)\\)$");
```

Comment:
I count one other instance of an itself pattern (in LockVisitor). The two patterns are distinct. Also, "itself" is meant to only be handled by the Lock Checker. So in this class, minimal handling of the "itself" expression is done - just enough to indicate to the caller that an "itself" expression was matched and that the caller needs to determine what expression "itself" refers to. So in the interest of leaving as little Lock Checker-specific code in this class as possible, I think the two should be kept separate.